pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -91,6 +91,7 @@ func parseConfig(config *zconfig.EdgeDevConfig, getconfigCtx *getconfigContext,
 		parseNetworkInstanceConfig(config, getconfigCtx)
 		parseContentInfoConfig(getconfigCtx, config)
 		parseVolumeConfig(getconfigCtx, config)
+		parseProfile(getconfigCtx, config)
```

Comment:
Where did you put the order comment?

Reply:
On top of parseProfile function, will mirror it here